presto-main/src/main/java/com/facebook/presto/sql/relational/optimizer/ExpressionOptimizer.java

```diff
@@ -56,6 +56,7 @@
 import static com.google.common.base.Predicates.instanceOf;
 import static com.google.common.collect.ImmutableList.toImmutableList;
 
+@Deprecated
```

Comment:
`ExpressionOptimizer` is for `RowExpression` only. It's just the name is confusing. Do not mark it as deprecated